recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
     - grib2.patch
 
 build:
-  number: 23
+  number: 24
```

Comment:
No need to bump the recipe build number for this.

Reply:
Reverted the bumping.